charts/wandb-base/README.md

```diff
@@ -197,32 +197,78 @@ In this example:
 - The `app` container will use the image `myapp:v1.0.0` with pull policy `Always`
 - The `sidecar` container will use the chart-level image `nginx:1.21.6` with pull policy `IfNotPresent`
 
+### Deployment and StatefulSet Labels and Annotations
+
+The chart supports adding custom labels and annotations to Deployment and StatefulSet resources. These can be defined at multiple levels, with the following precedence (highest to lowest):
+
+1. Local configuration (`deployment.labels` / `deployment.annotations` or `statefulset.labels` / `statefulset.annotations`)
+2. Global configuration (`global.deployment.labels` / `global.deployment.annotations` or `global.statefulset.labels` / `global.statefulset.annotations`)
```

Comment:
This will be helpful, but the main chart already defines `global.common.labels` and `global.common.annotations`. Can those be merged into the set of labels generated as well (inside the `"wandb-base.labels"` template)?